docs/UserGuide.md

```diff
@@ -199,7 +201,7 @@ and `johndoe@example.com` respectively.
 
 #### Finding a contact : `find`
 
-Finds persons whose names contain any of the given keywords. Also supports search with additional specifiers such as 
+Finds persons whose names contain any of the given keywords. Athena also supports search with additional specifiers such as 
```

Comment:
this adds inconsistency as all the other sentences do not have mention of Athena before the verb.

Reply:
i think you missed this!